frontend/packages/knative-plugin/src/utils/create-channel-utils.ts

```diff
@@ -36,10 +36,10 @@ export const getChannelKind = (ref: string): string => {
 
 export const useChannelList = (namespace: string): ChannelListProps => {
   const [accessData, setAccessData] = useSafetyFirst({ loaded: false, channelList: [] });
-  const channelResourcesList = getDynamicChannelModelRefs();
-  const accessList = [];
+  const channelResourcesList = useChannelResourcesList();
   React.useEffect(() => {
-    _.forIn(channelResourcesList, (channelRef: string) => {
+    const accessList = [];
+    _.forIn(channelResourcesList.channels, (channelRef: string) => {
```

Comment:
can we add the `.loaded` condition here and wrap for both forIn and promise in it.